9174 user research updates (#269)
CMSgov/bcda-static-site: _pages/api-documentation/get-a-bearer-token.md

BCDA protects its token endpoint with Basic Auth. Your credentials will be formatted as a client ID and client secret.
- **Sandbox environment**: Use a sample client ID and secret from the [sandbox credentials section]({{ '/api-documentation/get-a-bearer-token.html' | relative_url }}#sandbox-credentials).
- **Production environment**: Use the client ID and secret issued by your model-specific system during [production access]({{ '/production-access.html' | relative_url }}). Credentials must be rotated (renewed) every 90 days. 

### 2. Request a bearer token

<div class="usa-alert usa-alert--warning usa-alert--slim">
    <div class="usa-alert__body">
        <p class="usa-alert__text">Bearer tokens <a href="{{ '/api-documentation/get-a-bearer-token.html#troubleshooting' | relative_url }}">expire</a> 20 minutes after they are generated.</p>
    </div>
</div>

Start a request for a bearer token in your terminal window or using a tool such as Postman. The following example uses the credentials of the extra-small model entity in BCDA's sandbox environment:

#### Sample Credentials

> Example Client ID (Extra-Small Model Entities):

Reviewer: ```suggestion
Example client ID (Extra-Small Model Entities):
```

{% include copy_snippet.html code=site.data.credentials.sandbox.extra_small.client_id language="yaml" can_copy=true %}

> Example Client Secret (Extra-Small Model Entities):

Reviewer: ```suggestion
Example client secret (Extra-Small Model Entities):
```

{% include copy_snippet.html code=site.data.credentials.sandbox.extra_small.client_secret language="yaml" can_copy=true %}

#### Request header

> The header has `Authorization: Basic` followed by the credentials. Credentials (`clientID:secret`) are joined by a colon, then encoded in Base64 format.

Reviewer: ```suggestion
The header has `Authorization: Basic` followed by the credentials. Credentials (`client_id:client_secret`) are joined by a colon, then encoded in Base64 format.
```

#### Example curl command to request a bearer token

> The header has `Authorization: Basic` followed by credentials `{client_id}:{client_secret}`. Credentials are joined by a colon, then encoded in Base64 format. In this example, the authorization in the request header is replaced with the sample credentials.

Reviewer: I think we should go back to the previous copy, this is still not accurate.

Anytime a client asks BCDA for a token, the client must supply the header: `Authorization: Basic {client_id}:{client_secret}` where the `{client_id}:{client_secret}` are joined by a colon, then encoded in Base64 format. We should mention the required headers before the curl example like we do with other types of requests. Here is an example from 'How to access claims data": 
<img width="638" alt="image" src="https://github.com/user-attachments/assets/efac6804-079e-4bd4-9adf-621004761fd6" />
****

Reviewer: curl offers a convenient option in the `--user` flag that handles the Base64 encoding and `Authorization: Basic` header for the BCDA user. That's why Jen and I had separated the example request headers from the example curl commands in many of the other locations
